### ISSUE TITLE
Multiple bodies are not hidden (--reporter-htmlextra-hideResponseBody)

### ISSUE THREAD

**Issue by no-gi** [DannyDainton/newman-reporter-htmlextra#310]
**What Are You Seeing**

First of all - Great job!!!

Issue that I have is that the collection is containing multiple requests.

Some of them, not only one, are containing information in the request body I don't want to show in the report. For all other requests I want to show the request body.

So I used "--reporter-htmlextra-hideResponseBody" parameter for it as planed. I am currently strugely if I want to hide more the one request body, from a 2nd, 3rd ... call.

Do I have to use the parameter twice so for example:

--reporter-htmlextra-hideResponseBody "[NAME 1]" --reporter-htmlextra-hideResponseBody "[NAME 2]" --reporter-htmlextra-hideResponseBody "[NAME 3]"

Or do I have to use it only once, but for example separated with comma?

--reporter-htmlextra-hideResponseBody "[NAME 1],[NAME 2],[NAME 3"

I tested these different kinds already and it did not worked. So I created this issue here.

This brings me to the last question, is there an option to activate for example some debug log to get down to the root cause?
Maybe it's related to the structure in the collection I use: folders with sub folders. spaces in the folder and request names.

I use current version: 1.20.1 on Windows 10 64.

Thanks a lot in advance.

Best
Norman

**Comment by DannyDainton** [comment]
Hey @no-gi 

The syntax on the CLI should be this, to hide the **response** body:

```
newman run collection.json -r htmlextra --reporter-htmlextra-hideResponseBody "POST Raw Text,POST Form Data"
```

I've just tested this out with the Postman Echo collection and it seems to be doing what I expect it too. I hide the response bodies for the request names I mentioned in the command.

To hide the **request** body you would need to use this flag `--reporter-htmlextra-hideRequestBody`

**Comment by no-gi** [comment]
Hey Danny,



thanks a lot - it works fine.



Surprisingly, I tested this already before creation of the issue - no clue. Once more, thanks!



Best

Norman


Am 12. März 2021 um 13:38 schrieb Danny Dainton ***@***.***>:






Hey @no-gi


The syntax on the CLI should be this, to hide the response body:

newman run collection.json -r htmlextra --reporter-htmlextra-hideResponseBody "POST Raw Text,POST Form Data"


I've just tested this out with the Postman Echo collection and it seems to be doing what I expect it too. I hide the response bodies for the request names I mentioned in the command.

To hide the request body you would need to use this flag --reporter-htmlextra-hideRequestBody
